math/mathcore/inc/Math/Minimizer.h

```diff
@@ -171,18 +171,30 @@ class Minimizer {
    /// set initial covariance matrix
    virtual bool SetCovariance(std::span<const double> cov, unsigned int nrow);
 
-   /// set a new lower limit variable  (override if minimizer supports them )
-   virtual bool SetLowerLimitedVariable(unsigned int  ivar , const std::string & name , double val , double step , double lower ) {
-      return SetLimitedVariable(ivar, name, val, step, lower, std::numeric_limits<double>::infinity() );
+   /// set a new lower limit variable  (override if minimizer supports them), leave upper bound unlimited
```

Comment:
```suggestion
   /// Set a new lower limit variable  (override if minimizer supports them), leave upper bound unlimited.
```
If you already change the line, you can also follow the convention of starting the doxygen brief summaries with a capital letter and ending them with a period.